go.sum

```diff
@@ -1098,6 +1096,8 @@ github.com/inconshreveable/mousetrap v1.0.0 h1:Z8tu5sraLXCXIcARxBp/8cbvlwVa7Z1NH
 github.com/inconshreveable/mousetrap v1.0.0/go.mod h1:PxqpIevigyE2G7u3NXJIT2ANytuPF1OarO4DADm73n8=
 github.com/influxdata/influxdb v0.0.0-20190411212539-d24b7ba8c4c4/go.mod h1:qZna6X/4elxqT3yI9iZYdZrWWdeFOOprn86kgg4+IzY=
 github.com/influxdata/influxdb v1.7.7/go.mod h1:qZna6X/4elxqT3yI9iZYdZrWWdeFOOprn86kgg4+IzY=
+github.com/itdove/idp-client-api v0.0.0-20211005141937-61a3da19b015 h1:oParYSk6HIlFbKmRKoYr/L+wtSdvpJJycv/d/zxpDpQ=
```

Comment:
Don't we need to merge idp-client-api first and not pull from itdove?

Reply:
yes sorry... to tired